todo/admin.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib import admin
+
+from todo.models import Tag, Task
+
+
+@admin.register(Task)
+class TaskAdmin(admin.ModelAdmin):
+    pass
+
+
+@admin.register(Tag)
+class AdminTag(admin.ModelAdmin):
```

Comment:
you called this class `AdminTag `whereas the one above is `TaskAdmin `- stick up yo one naming style. More common for django is to call `ModelnameAdmin`, like `TagAdmin`